automation/features/step-definitions/tranches.steps.js

```diff
@@ -0,0 +1,28 @@
+const { Given, When, Then } = require("@wdio/cucumber-framework");
+const TranchesPage = require("../pageobjects/tranches.page");
+
+When(/^I click on the first tranche$/, () => {
+  const tranche = TranchesPage.trancheLink;
+  tranche.click();
+});
+
+Then(/^First tranche contains unlocked tokens$/, () => {
+  const unlockedTokens = TranchesPage.firstTrancheUnlockedTokens;
+
+  let unlockedNum = Number(unlockedTokens.getText().replace(/,/g, ""));
+  expect(unlockedNum).toBeGreaterThan(0);
+});
+
+Then(/^I can see tranches are displayed$/, () => {
+  const displayedTranche = browser.getByTestId(TranchesPage.trancheTestId);
```

Comment:
I think `TranchesPages.trancheTestId` will return undefined unless you either create an instance of the TranchesPage class or make `trancheTestId` a static property

Reply:
this is not a running/written test and will be addressed in other pr's once we come to write tranches tests